### PR TITLE
Document verbose-debug

### PR DESCRIPTION
**Pull request by Malabarba**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1012,6 +1012,15 @@ using the debugger.
...
 Check the value of `nrepl-log-messages`. It should be non-nil.
 
+### `cider-debug` complains that it “failed to instrument ...”
+
+In the repl buffer, issue the following.
+
+    your.namespace> (ns cider.nrepl.middleware.util.instrument)
+    cider.nrepl.middleware.util.instrument> (def verbose-debug true)
+
+This will cause cider to print extensive information on the `nrepl` buffer when you try to debug an expression (e.g., with `C-u C-M-x`). [File an issue](https://github.com/clojure-emacs/cider-repl/issues/new) and copy this information.
```
Comment (expez) [inline README.md]: If you read this as a nublet, will you know what the `nrepl` buffer is?

Comment (Malabarba) [inline README.md]: @expez Reworded

Comment (bbatsov) [inline README.md]: The buffer is actually named `*nrepl-server*`.

File: README.md [hunk 2]
```diff
@@ -1012,6 +1012,15 @@ using the debugger.
 
 Check the value of `nrepl-log-messages`. It should be non-nil.
 
+### `cider-debug` complains that it “failed to instrument ...”
+
+In the repl buffer (the one called `*cider-repl PROJEC-NAME*`), issue the following.
```
Comment (bbatsov) [inline README.md]: We capitalise REPL in the rest of the readme.

**Comment by Malabarba** [comment]
Sorry about the confusion. It was only REPL buffer all along. Is the current version ok or should I specify what the repl buffer is?

**Comment by expez** [comment]
Looks good now!

**Pull request merged**